### DISCUSSION TITLE
WIP- Kanban & Calendar Views [Mar 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1584] [Discord Threads]
> 🙋‍♂️ **ike5**  
## Hello NocoDB team, I used the 1-click Heroku install and connected to my own MySQL database, but I don't see any option for the Kanban nor the Calendar. Anyone experiencing the same thing?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/956693318086066196/Screen_Shot_2022-03-24_at_4.15.15_PM.png)

**Comment by dstala** [comment]
> 👨‍🦲 **dstala**  
#### ike5@discord Thanks for trying out NocoDB!    
#### Kanban & Calendar views are work in progress. Currently NocoDB supports Grid, Gallery & Form Views only.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/956693318765527061)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by ppkliu** [reply]
How to enable kanban view in development mode?
I can run some testing with kanban view.
Hope to speed up the development of kanban view.
